tests/Aquifer.Public.API.IntegrationTests/BiblesListTestFixture.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Net;
+using Aquifer.Public.API.Endpoints.Bibles.List;
+using FastEndpoints.Testing;
+using FastEndpoints;
+
+namespace Aquifer.Public.API.IntegrationTests;
+
+public sealed class BiblesListTestFixture(AppFixture _appFixture) : TestBase<AppFixture>
+{
+    [Fact]
+    public async Task ValidRequest_ShouldReturnSuccess()
```

Comment:
This isn't the most ideal test but my goal in this PR is to demonstrate that the integration tests work.  I can come back later and write some better tests for the API endpoints if desired.